### ISSUE TITLE
Activity: some actions are not accessible on the activity details screen

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12853]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to Activity and tap on any activity to see details.
1. Repeat step 2 for several different types of activities.

**Result:** some activities such as "Post modified" or "Post draft modified" are inaccessible in the activity detail while other activities such as "Video uploaded" and "Image attached" can be accessed. ([1m30s](https://cloudup.com/cd5qkJbh66R))

<img src="https://user-images.githubusercontent.com/1119271/68170249-443cf400-ff2c-11e9-94a2-d6e9d06aa51e.png" width="270" alt="activity-detail-action-is-not-accessible">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
